### DISCUSSION TITLE
Testing Tools

### DISCUSSION THREAD

**Discussion by rajgits** [movementlabsxyz/movement#349] [General]
can anybody suggest some good Testing tool for MOVE

**Comment by l-monninger** [comment]
What kind of testing? The CLI/compiler has good tooling for unit testing. For, integration and e2e, spinning up a local network is often your best bet.

**Reply by rajgits** [reply]
Okay basically for TON i am using https://ide.nujan.io/ these online IDE for immediate result. so expecting some kind of those.

**Reply by l-monninger** [reply]
Gotcha. Yeah, I'm personally not an advocate for these kinds of bells and whistles. But, I think @0xPrimata enjoys this sort of experience and has had a few conversations to this end.
